### PR TITLE
Creates views, controllers and routes for registered users and pending registrations, updates links on index.html

### PR DESCRIPTION
**Pull request by bryanarboledaa**
Created views for Pending registrations and Registered users. Also updated AdminController and routes to be pages able to work. 

**Screenshots**

image

image

image

**Note:**

Will style UI after ActionMailer works

### CODE REVIEW

File: app/controllers/admin_portals_controller.rb [hunk 1]
```diff
@@ -34,4 +35,13 @@ def create_user
     @user.save
     redirect_to admins_home_path if @user.save
   end
+
+  def show_pending_users
+    @users = User.all
```
Comment (brandomoreno06) [inline app/controllers/admin_portals_controller.rb]: I think, to show the pending users, we should get all those with status "pending".

File: app/views/admin_portals/show_pending_users.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <% if user.status == 'pending' %>
+    <%= user.username %> </br>
+    <%= user.email %> </br>
+    <%= user.status %> </br>
+    <%= link_to "View", admins_view_path(id: user.id), method: 'get' %>
+    <%= link_to "Edit User", edit_user_path(id: user.id), method: 'get' %>
+    <%= button_to "Approve", admins_manage_path(id: user.id), method: 'put' %>  
+    <%= link_to "Back", admins_home_path %>  
+  <% end %>
+<% end %>
```
Comment (brandomoreno06) [inline app/views/admin_portals/show_pending_users.html.erb]: Please add a new line at the end of the file.

File: app/views/admin_portals/show_registered_users.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<% @users.each do |user| %>
+  <% if user.status == 'approved' %>
+   <p> Username: <%= user.username %> </p>
+   <p> Email: <%= user.email %>  </p>
+    <%= link_to "View", admins_view_path(id: user.id), method: 'get' %>
+    <%= link_to "Edit User", edit_user_path(id: user.id), method: 'get' %> </br>
+  <% end %>
+<% end %>
+
+<%= link_to "Back", admins_home_path %>
```
Comment (brandomoreno06) [inline app/views/admin_portals/show_registered_users.html.erb]: Please add a new line at the end of the file.

File: app/controllers/admin_portals_controller.rb (hunk 1 above)
Comment (bryanarboledaa) [reply app/controllers/admin_portals_controller.rb]: Got it.

**Review by brandomoreno06 — APPROVED** [review]
This is okay now. Approved. Cheers!

**Pull request merged**